### ISSUE TITLE
[TLSF] remove duplicates

### ISSUE THREAD

**Issue by 5nizza** [SYNTCOMP/benchmarks#25]
From issue #5

> Also detector is equivalent to ltl2dba_C2, up to renaming of atomic propositions

https://github.com/SYNTCOMP/benchmarks/blob/master/tlsf/ltl2dba/parametric/ltl2dba_C2.tlsf
https://github.com/SYNTCOMP/benchmarks/blob/master/tlsf/detector/parametric/detector.tlsf

(that is indeed the case)

TODO: remove from the release at least.
Long-term: remove _all_ duplicates.

**Comment by 5nizza** [comment]
remark: check:
- there are similarities between collector benchmarks and ltl2dba
- between ltl2dba parametric and non-parametric ltl2dba

**Comment by 5nizza** [comment]
for 2023, I remove detector from the release while keeping it in the repo.

**Comment by adl** [comment]
The following CSV files contain a list of 45 duplicate models I found in the `tlsf/` directory, and 163 duplicates from the `tlsf-fin/` subdirectory.

[tlsf-dups.csv](https://github.com/user-attachments/files/17058427/tlsf-dups.csv)
[tlsf-fin-dups.csv](https://github.com/user-attachments/files/17058433/tlsf-fin-dups.csv)

To find those, I used `syfco` to convert each tlsf model into an LTL formula with `syfco -m fully -f ltlxba` or `syfco -m fully -f ltlxba-fin`.   If the formula produced from file CURFILE is equal to the formula of a previously processed file PREVFILE, the CSV files contains a line of the form "DUP,CURFILE,PREVFILE".

If the formula isn't equal to any previously seen formula, I also tried to use `ltlfilt` to relabel all atomic propositions in the order they are discovered (the development version of `ltlfilt` has option to do such relabeling while preserving the I/O nature of each atomic proposition).   If the relabeled formula is equal to a previously relabeled formula the CSV files contains a line of the form "REL,CURFILE,PREVFILE".   This detection of duplicate-up-to-a-relabeling formulas isn't a perfect isomorphism check, but it still catch additional some pairs.

The competition could save time by skipping/erasing all models from that CURFILE column, since those models are already tested.

The quick and dirty script I used to execute the above is [checkdups.py.gz](https://github.com/user-attachments/files/17058614/checkdups.py.gz), but it requires the development version of Spot (or Spot 2.13 if that has been released by the time your read this).
